install.sh

```diff
@@ -18,15 +18,27 @@ set -euo pipefail
   at_exit() {
     ret=$?
     if [[ $ret -gt 0 ]]; then
-      log "the script failed with error $ret.\n" \
-        "\n" \
-        "To report installation errors, submit an issue to\n" \
-        "    https://github.com/direnv/direnv/issues/new/choose"
+      log "the script failed with error ${ret}.\n" \
+      "To report installation errors, submit an issue to\n" \
+      "https://github.com/direnv/direnv/issues/new/choose"
     fi
     exit "$ret"
   }
   trap at_exit EXIT
 
+  if [ $# -ge 1 ];then
+    if [ "${1}x" == "--helpx" ] || [ "${1}x" == "-hx" ];then
```

Comment:
It's an historic artifact and shells from this century should no longer need this hack:

```suggestion
    if [ "${1}" == "--help" ] || [ "${1}" == "-h" ]; then
```